src/pymp4/parser.py

```diff
@@ -84,22 +84,22 @@ def _sizeof(self, context, path):
 
 FileTypeBox = Struct(
     "type" / Const(b"ftyp"),
-    "major_brand" / PaddedString(4, "utf8"),
+    "major_brand" / PaddedString(4, "ascii"),
```

Comment:
Why replace explicitly "utf8" encodings with ascii? This is done various times in this commit and likely shouldn't be.